ccd-definition/ComplexTypes.json

```diff
@@ -44,15 +44,15 @@
   },
   {
     "LiveFrom": 42736,
-    "ID": "Children",
+    "ID": "ChildrenType",
```

Comment:
When seeing `ChildrenType` I expect some sort of enumerated children classification like `POOR` or `RICH`. Think of complex type ID as of class name in Java world. Since that complex type describes child perfect name for complex type would be `Child` and then `Children` would be:

```
 - firstChild: Child
 - additionalChildren: List<Child>
```

Reply:
This is still outstanding. Please change `ChildrenType` to `Child`